perf/intkey_workload/src/main.rs

```diff
@@ -28,9 +28,13 @@ mod intkey_addresser;
 mod intkey_iterator;
 mod intkey_transformer;
 
+use std::convert::From;
```

Comment:
I don't think this has to be imported

Reply:
rustc doesn't warn about an unneeded import.